src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -876,7 +876,12 @@ class CommonVisitor : public AST::BaseVisitor<Struct> {
         }
 
         if( !type && raise_error ) {
-            throw SemanticError("Unsupported type annotation: " + var_annotation, loc);
+            if (var_annotation == "int") {
+                throw SemanticError(var_annotation + " type is not supported yet. " , loc);
```

Comment:
Add a hint: "hint: use i8, i16, i32 or i64 for now". Lookup other place in the code how to add a hint.

Reply:
Please review the changes I have made.